doc/user/content/sql/system-catalog/_index.md

```diff
@@ -12,6 +12,12 @@ menu:
 disable_list: true
 ---
 
+{{< warning >}}
+Views that directly reference these objects cannot include `NATURAL JOIN` or
+`*` expressions. Instead, Expand all wildcards and convert all `NATURAL JOIN`s
```

Comment:
```suggestion
`*` expressions. Instead, project the required columns and convert all `NATURAL JOIN`s
```

Reply:
The upper-case E feels like a personal attack, @jkosh44. 😂

Reply:
Oy, that's what I get for copy and pasting.